tutorials/speaker_tasks/Speaker_Diarization_Inference.ipynb

```diff
@@ -376,15 +376,15 @@
             "metadata": {},
             "outputs": [],
             "source": [
-                "pretrained_vad = 'vad_marblenet'\n",
+                "pretrained_vad = 'vad_multilingual__marblenet'\n",
```

Comment:
You need to update vad post processing params in notebook, there is no silence detected with this model with previous settings

Reply:
@nithinraok  yeah it's a typo which loads old model. Should be okay for now.

Reply:
Did you check the output of VAD on an4 sample?

Reply:
yep